api-gateway/src/test/java/com/petclinic/bffapigateway/domainclientlayer/VisitsServiceClientIntegrationTest.java

```diff
@@ -55,6 +56,19 @@ static void tearDown() throws IOException {
         server.shutdown();
     }
 
+    @Test
+    void getAllVisits() throws JsonProcessingException {
+        final VisitResponseDTO visitResponseDTO = new VisitResponseDTO("773fa7b2-e04e-47b8-98e7-4adf7cfaaeee", 2023, 11, 20, "test visit", 1, 1, false);
+        final VisitResponseDTO visitResponseDTO2 = new VisitResponseDTO("73fa7b2-e04e-47b8-98e7-4adf7cfaaee", 2023, 11, 20, "test visit", 1, 1, false);
+        server.enqueue(new MockResponse().setHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                .setBody(objectMapper.writeValueAsString(Arrays.asList(visitResponseDTO, visitResponseDTO2))).addHeader("Content-Type", "application/json"));
+
+        Flux<VisitResponseDTO> visitResponseDTOFlux = visitsServiceClient.getAllVisits();
+        StepVerifier.create(visitResponseDTOFlux)
+                .expectNext(visitResponseDTO)
+                .expectNext(visitResponseDTO2)
+                .verifyComplete();
+    }
```

Comment:
Service Client Integration test is reactive, StepVerifier could check the attributes of visitResponseDTO and visitResponseDTO2 using assertEquals.